packages/office-ui-fabric-react/src/components/DetailsList/__snapshots__/DetailsHeader.test.tsx.snap

```diff
@@ -889,6 +889,7 @@ exports[`DetailsHeader can render a hidden select all checkbox in single selecti
         line-height: 32px;
         min-width: 100%;
         padding-bottom: 1px;
+        padding-top: 1px;
```

Comment:
Was this supposed to be part of this change?

Reply:
I had screener failing before running update-snapshots.

Reply:
@ecraig12345 Merging master fixed this.